.travis-scripts/verify_storing_counterexamples.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# Verify that we can configure storing counterexamples
+#
+
+mix test test/store_counter_examples_test.exs --include will_fail:true && exit 1
+mix test test/verify_counter_examples_test.exs
```

Comment:
That's a clever idea to provide a separate build task.

Reply:
I actually got that from the Travis documentation. :)